config/database.yml

```diff
@@ -6,8 +6,7 @@ default: &default
 development:
   <<: *default
   database: engsi_database_development
-  username: postgres
-  password: 
+
```

Comment:
Não seria bom deixar o `username: postgres` e `password:  ` pra manter o travis e pra quando a gente for fazer o pull saber onde mudar?

Reply:
Seria sim, errei pra subindo esse arquivo